mne/preprocessing/eog.py

```diff
@@ -61,7 +63,7 @@ def find_eog_events(raw, event_id=998, l_freq=1, h_freq=10,
                                   filter_length=filter_length,
                                   tstart=tstart)
     # Map times to corresponding samples.
-    eog_events[:, 0] = times[eog_events[:, 0] - raw.first_samp]
+    eog_events[:, 0] = np.round(times[eog_events[:, 0] - raw.first_samp])
```

Comment:
`np.round` returns `float` type so this still needs `.astype(int)` to be fully explicit (and avoid potential NumPy truncation warnings down the line)